apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000004_add_input_data_to_dataset_item.sql

```diff
@@ -0,0 +1,48 @@
+--liquibase formatted sql
+--changeset thiagohora:add_input_data_to_dataset_item
+
+ALTER TABLE ${ANALYTICS_DB_DATABASE_NAME}.dataset_items
+    ADD COLUMN IF NOT EXISTS input_data Map(String, String) DEFAULT map();
+
+INSERT INTO ${ANALYTICS_DB_DATABASE_NAME}.dataset_items
```

Comment:
I don't think this should be backfilled in a migration as we don't control many of the deployment environments. It's related to my other comments, we should better define the resolution logic if there's data in the old and the new fields. Data migrations should be tackled separately.

Reply:
For the cloud we would need to handle it anyway, and I believe not offering this may upset clients. I don't know, just a feeling

Reply:
Agreed to tackle the data migration separately, as this change handles it.